docs: refactor ip-restriction.md (#7402)
apache/apisix: docs/zh/latest/plugins/ip-restriction.md

## 描述

`ip-restriction` 插件可以通过将 IP 地址列入白名单或黑名单来限制对服务或路由的访问。

> 支持对单个 IP 地址，多个 IP 地址和类似 `10.10.10.0/24` 的 CIDR（Classless InterDomain Routing，无类域间路由）范围的限制。

Reviewer: ```suggestion
支持对单个 IP 地址、多个 IP 地址和类似 `10.10.10.0/24` 的 CIDR（Classless InterDomain Routing，无类域间路由）范围的限制。
```

> 支持对单个 IP 地址、多个 IP 地址和类似 `10.10.10.0/24` 的 CIDR（Classless InterDomain Routing，无类域间路由）范围的限制。

Reviewer 2: ```suggestion
支持对单个 IP 地址、多个 IP 地址和类似 `10.10.10.0/24` 的 CIDR（无类别域间路由）范围的限制。
```

Author: Hi, "CIDR" is not as common as "IP" or other terms. From the user's perspective and refer to the [doc style guide](https://apisix.apache.org/zh/docs/general/documentation-style-guide/), I prefer to keep the full English name.

Reviewer 2: Hi, the full English name doesn't make sense to Chinese readers, right? 😉 Also, if readers want to use the full name, just searching the CIDR.

![image](https://user-images.githubusercontent.com/2106987/177950478-8efff267-3077-4a72-8d8e-021be8539074.png)

}'
```

> 当访问未允许的 IP 时，默认返回 `{"message":"Your IP address is not allowed"}`。如果想使用自定义的 `message`，可以在插件部分进行配置：

Reviewer 2: ```suggestion
当使用白名单之外的 IP 访问时，默认返回 `{"message":"Your IP address is not allowed"}`。如果想使用自定义的 `message`，可以在插件配置中进行调整：
```

## 测试插件

> 启用插件后，使用 `curl` 命令先从 IP 地址 `127.0.0.1` 发出请求：

Reviewer 2: ```suggestion
启用插件后，使用 `curl` 命令访问 APISIX 实例地址：
```

```shell
curl http://127.0.0.1:9080/index.html -i
```

返回 200 HTTP 状态码，代表访问成功：

```shell
HTTP/1.1 200 OK
...
```

再从 IP 地址 `127.0.0.2` 发出请求：

```shell
curl http://127.0.0.1:9080/index.html -i --interface 127.0.0.2
```

返回 403 HTTP 状态码，代表访问被阻止：

```shell
HTTP/1.1 403 Forbidden
...
{"message":"Your IP address is not allowed"}
```

> 要更改列入白名单或黑名单的 IP 地址，你只需更新插件配置，无需重启服务：

Reviewer 3: ```suggestion
如果你需要更改白名单或黑名单的 IP 地址，你只需更新插件配置，无需重启服务：
```
